analysis/dataset_definition_pf_data_development.py

```diff
@@ -19,21 +21,31 @@
     "pharm_first_service": ["983341000000102"],
 }
 
-pharmacy_first_codes = [
-    code for codelist in pharmacy_first_dict.values() for code in codelist
+pharmacy_first_event_codes = [
+    code for codelist in pharmacy_first_events_dict.values() for code in codelist
 ]
 
+pharmacy_first_med_codes = (
+    acute_otitis_media_cod
```

Comment:
See change to codelists.py
```suggestion
    acute_otitis_media_treatment_cod
```

Reply:
thanks :) addressed in ca1e1573c933a290905a92e8d8cc6065fe151a77